core/src/main/java/org/mskcc/cbio/oncokb/util/AlterationUtils.java

```diff
@@ -1327,6 +1370,141 @@ private static List<Alteration> findAlterationsByStartWith(String startWith, Lis
         return matchedAlterations.stream().collect(Collectors.toList());
     }
 
+    public static Alteration findExactlyMatchedAlteration(ReferenceGenome referenceGenome, Alteration alteration, List<Alteration> fullAlterations) {
```

Comment:
The following methods are from AlterationBoImpl. But please review lines 1416-1426.